Makefile

```diff
@@ -6,7 +6,7 @@ OS := $(shell uname -s)
 INTERACTIVE := $(shell [ -t 0 ] && echo 1)
 
 # BUILD_DRY_RUN determines the value of the --dry-run flag of the build command. Should be 'true' or 'false'.
-BUILD_DRY_RUN := true
+BUILD_DRY_RUN ?= true
```

Comment:
Was this not being overridden with `:=`?

Reply:
it may be working. let me test it and confirm.